src/gobstuf/rest/brp/argument_checks.py

```diff
@@ -76,11 +76,11 @@ class ArgumentCheck():
         }
     }
 
-    is_valid_gemeente = {
-        'check': validate_gemeente,
+    is_valid_gemeentecode = {
+        'check': validate_gemeentecode,
         'msg': {
             "code": "invalidGemeente",
-            "reason": "Waarde is geen geldige gemeente",
+            "reason": "Waarde komt niet voor in de tabel.",
```

Comment:
Maybe change this to "Waarde is geen geldige gemeentecode"? "de tabel" may be a bit confusing for API users.

Reply:
This was described in the ticket, will discuss with Onno or Rob

Reply:
Changed